tools/StaticAnalysis/Exceptions/Az.Storage/BreakingChangeIssues.csv

```diff
@@ -38,3 +38,120 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll","Microsoft.Azure.Commands.Management.Storage.GetAzureStorageAccountManagementPolicyCommand","Get-AzStorageAccountManagementPolicy","0","3000","The type of property 'Kind' of type 'Microsoft.Azure.Commands.Management.Storage.Models.PSStorageAccount' has changed from 'System.Nullable`1[Microsoft.Azure.Management.Storage.Models.Kind]' to 'System.String'.","Change the type of property 'Kind' back to 'System.Nullable`1[Microsoft.Azure.Management.Storage.Models.Kind]'."
 "Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll","Microsoft.Azure.Commands.Management.Storage.RemoveAzureStorageAccountManagementPolicyCommand","Remove-AzStorageAccountManagementPolicy","0","3000","The type of property 'Kind' of type 'Microsoft.Azure.Commands.Management.Storage.Models.PSStorageAccount' has changed from 'System.Nullable`1[Microsoft.Azure.Management.Storage.Models.Kind]' to 'System.String'.","Change the type of property 'Kind' back to 'System.Nullable`1[Microsoft.Azure.Management.Storage.Models.Kind]'."
 "Microsoft.Azure.PowerShell.Cmdlets.Storage.Management.dll","Microsoft.Azure.Commands.Management.Storage.SetAzureStorageAccountManagementPolicyCommand","Set-AzStorageAccountManagementPolicy","0","3000","The type of property 'Kind' of type 'Microsoft.Azure.Commands.Management.Storage.Models.PSStorageAccount' has changed from 'System.Nullable`1[Microsoft.Azure.Management.Storage.Models.Kind]' to 'System.String'.","Change the type of property 'Kind' back to 'System.Nullable`1[Microsoft.Azure.Management.Storage.Models.Kind]'."
+"Microsoft.Azure.PowerShell.Cmdlets.Storage.dll","Microsoft.WindowsAzure.Commands.Storage.Queue.GetAzureStorageQueueCommand","Get-AzStorageQueue","0","3000","The type of property 'CloudQueue' of type 'Microsoft.WindowsAzure.Commands.Common.Storage.ResourceModel.AzureStorageQueue' has changed from 'Microsoft.WindowsAzure.Storage.Queue.CloudQueue' to 'Microsoft.Azure.Storage.Queue.CloudQueue'.","Change the type of property 'CloudQueue' back to 'Microsoft.WindowsAzure.Storage.Queue.CloudQueue'."
```

Comment:
What changes will we require in the migration guide for this?

Reply:
I will add a issue to track the migration Guild. 
I will provide all information for the Guild, So OneSDK team can help to write it.